src/uct/api/v2/uct_v2.h

```diff
@@ -174,6 +177,13 @@ typedef struct {
      * This field is set by the UCT layer.
      */
     ucs_linear_func_t   latency;
+
+    /**
+     * Maximum number of endpoints which could be utilized simultaneously for
+     * posting operations.
```

Comment:
Approximated maximum number of endpoints that could have outstanding operations simultaneously.
If additional endpoints are being used for sending operations, these operations will most likely return UCS_ERR_NO_RESOURCE.
Protocols that require sending to multiple destinations at the same time (such as keepalive) could benefit
from using a transport that has a large number of maximum inflight endpoints.